rpc/rpcserver/server.go

```diff
@@ -1267,8 +1271,8 @@ func (t *ticketbuyerServer) StopAutoBuyer(ctx context.Context, req *pb.StopAutoB
 	*pb.StopAutoBuyerResponse, error) {
 
 	err := t.loader.StopTicketPurchase()
-	if err == loader.ErrTicketBuyerStopped {
-		return nil, grpc.Errorf(codes.FailedPrecondition, "Ticket buyer is not running")
```

Comment:
Missed that, restored. Also added the translation to `StartAutoBuyer` above. Thanks.